dev-docs/bidders/glimpse.md

```diff
@@ -1,95 +1,137 @@
+  async
+  src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"
+></script>
 
-## Setup Guide
+<script
+  async
+  src="https://<PUBLISHER_DOMAIN>/prebid.js"
+></script>
+```
 
-Follow these steps to configure and add the glimpse module to your Prebid.js integration.
+### Publisher Registration
+
+Coming soon.
 
-### 0. Preconditions
+### Domain Registration
 
-- A built prebid module with the glimpse adaptor included
-- You've included the built prebid adaptor and GPT script in your websites html code
-- You've setup GAM mappings
+Coming soon.
 
-### 1. Create an account and setup your domain via the Publisher Portal
+### Ad Unit Registration
 
 Coming soon.
 
-### 2. Enable Glimpse as a bidder on your ad units
+## Configuration
+
+### Bid Requests
+
+Our adapter captures the following values in the `params` block of each bid request:
+
+| Name          | Scope    | Type   | Description                                                                                                      | Example                |
+| ------------- | -------- | ------ | ---------------------------------------------------------------------------------------------------------------- | ---------------------- |
+| `placementId` | Required | string | A unique identifier associated with a publisher and ad unit, provided by the Glimpse Publisher Portal when registering a unit | 'glimpse-placement-id' |
+| `keywords` | Optional | Record<string, any> | Audience targeting data (applies to the single ad unit) | { sticky: true } |
+
+#### Example
 
 ```javascript
-const adUnits = {
-  code: 'your-ad-unit-div-id',
-  mediaTypes: {
-    banner: {
-      sizes: [[300, 250]],
+const units = [{
+    code: "ad-unit-0",
+    mediaTypes: {
+        banner: { sizes: [[300, 250]] },
     },
-  },
-  bids: [
-    {
-      bidder: 'glimpse',
-      params: {
-        placementId: 'placementId-from-publisher-portal',
+    bids: [{
+        bidder: "glimpse",
+        params: {
+            placementId: "glimpse-placement-id",
+            keywords: {
+              sticky: true,
+            },
+        },
+    }],
+}]
+```
+
+### Bidder Config
+
+Our adapter captures the following values in the `glimpse` block of the `BidderConfig` object:
+
+| Name          | Scope    | Type   | Description                                                                                                      | Example                |
+| ------------- | -------- | ------ | ---------------------------------------------------------------------------------------------------------------- | ---------------------- |
+| `demo` | Optional | boolean | Triggers Glimpse to respond with dummy bids (defaults to false) | true |
+| `account` | Optional | number | Your Glimpse account id | 1234 |
+| `demand` | Optional | string | The target demand source (defaults to 'glimpse') | 'xandr' |
+| `keywords` | Optional | Record<string, any> | Audience targeting data (applies to all ad units) | { sticky: true } |
```

Comment:
keywords and audience targeting data must be accepted on the standard locations as described in https://docs.prebid.org/features/firstPartyData.html

I checked your adapter code and it's not referring to `ortb2`. Please confirm that you agree to conform to this convention within a few weeks and then I'll merge these docs.

For example, publishers should be able to set cross-bidder first party data like this and have all adapters deal with it rather than having to pass such data individually into separate adapters.

```
pbjs.setConfig({
    ortb2: {
        site: {
            keywords: "sports, basketball",
            ext: {
                data: {
                    sticky: true
                }
            }
        }
    }
});
```

Reply:
Thanks! Yeah we'll be sure to update that.